astroquery/mast/cutouts.py

```diff
@@ -314,3 +314,228 @@ def get_cutouts(self, coordinates=None, size=5, sector=None, objectname=None):
+
+        # If radius is just a number we assume degrees
+        if isinstance(radius, float):
+            radius = Angle(radius, u.deg)
+            params["radius"] = radius.deg
+
+        response = self._service_api_connection.service_request_async("survey", params)
+        response.raise_for_status()  # Raise any errors
+
+        survey_json = response.json()['surveys']
+
+        if not len(survey_json):
+            warnings.warn("Coordinates are not in any Z survey.", NoResultsWarning)
+        return survey_json
+
+    def download_cutouts(self, coordinates, size=5, units="px", survey=None, form="fits", path=".", **img_params):
+        """
+        Download cutout target pixel file(s) around the given coordinates with indicated size.
+
+        Parameters
+        ----------
+        coordinates : str or `astropy.coordinates` object
+            The target around which to search. It may be specified as a
+            string or as the appropriate `astropy.coordinates` object.
+            One and only one of coordinates and objectname must be supplied.
+        size : int, array-like, `~astropy.units.Quantity`
+            Optional, default 5 pixels.
+            The size of the cutout array. If ``size`` is a scalar number or
+            a scalar `~astropy.units.Quantity`, then a square cutout of ``size``
+            will be created.  If ``size`` has two elements, they should be in
+            ``(ny, nx)`` order.  Scalar numbers in ``size`` are assumed to be in
+            units of pixels. `~astropy.units.Quantity` objects must be in pixel or
+            angular units.
+        units : str
+            Optional, default is px.
+            The `units` parameter accepts the following units: d (degrees),
+            s (arcseconds), m (arcminutes), and px (pixels).
+        survey : str
+            Optional
+            The survey to restrict the cutout. The survey parameter will restrict to
+            only the matching survey. Default behavior is to return all matched surveys.
+        form : str
+            Optional
+            The cutout file format. Default is fits, valid options are fits, jpg, png.
+        path : str
+            Optional.
+            The directory in which the cutouts will be saved.
+            Defaults to current directory.
+        **img_params : dict
+            Optional, only used if format is jpg or png
+            Dictionary of image parameters.
+            Valid entries are:
+                stretch : The stretch to apply to the image array, default asinh. Valid values are:
+                    asinh, sinh, sqrt, log, linear.
+                minmax_percent : Percent interval of pixels to keep when scaling,
+                    default is [0.5,99.5]
+                minmax_value : Pixel value interval of pixels to keep when scaling, default is
+                    to use minmax_percent instead
+                invert : True/false, if True the image will be inverted
```

Comment:
This *is* a dictionary, but the user doesn't enter it that way, so you want to write it up differently.  Also you don't have to go into all the detail here, you can just say "Valid parameters are stretch, minmax_percent, minmax_value, and invert. These arguments are documented here: https://astrocut.readthedocs.io/en/latest/api/astrocut.img_cut.html"

Here is an example:
```
        **criteria
            Criteria to apply. At least one non-positional criteria must be supplied.
            Valid criteria are coordinates, objectname, radius (as in `query_region` and `query_object`),
            and all observation fields returned by the ``get_metadata("observations")``.
            The Column Name is the keyword, with the argument being one or more acceptable values for that parameter,
            except for fields with a float datatype where the argument should be in the form [minVal, maxVal].
            For non-float type criteria wildcards maybe used (both * and % are considered wildcards), however
            only one wildcarded value can be processed per criterion.
            RA and Dec must be given in decimal degrees, and datetimes in MJD.
            For example: filters=["FUV","NUV"],proposal_pi="Ost*",t_max=[52264.4586,54452.8914]
```